romancal/pipeline/exposure_pipeline.py

```diff
@@ -62,6 +62,8 @@ def process(self, input):
         log.info('Starting Roman exposure calibration pipeline ...')
         if isinstance(input, str):
             input_filename = basename(input)
+        else:
+            input_filename = None
```

Comment:
I'm not sure that is what we want here. That will later set result.meta.filename to None. I think we want
`input_filename = input.meta.filename `
in the case we are passing a model.

Reply:
If passing a model, input.meta.filename is already set

Reply:
And I think its only re-set if its not None, on line 75

Reply:
Oops my error, you are correct.